### PR TITLE
correction

### PR DESCRIPTION
**Pull request by Cilla-luodan**

### CODE REVIEW

File: markdown/Flutter NG/Misc/Electron 410 发版说明/视频.md [hunk 1]
```diff
@@ -52,38 +52,46 @@ SDK 默认使用播放设备为声卡采集设备，自该版本起，你可以
 该版本新增了如下适用于空间音效场景的特性，在虚拟互动场景下可以有效提升用户的临场感体验。
 
 - 隔声区域：你可以通过调用 `setZones` 设置隔声区域和声音衰减系数。当音源 (可以为用户或媒体播放器) 跟听声者分属于音障区域内部和外部时，会体验到类似真实环境中声音在遇到建筑隔断时的衰减效果。你也可以通过调用 `setPlayerAttenuation` 和 `setRemoteAudioAttenuation` 方法分别针对媒体播放器和用户设置声音衰减属性，并指定是否使用该设置强制覆盖 `setZones` 中的声音衰减系数。
-- 多普勒音效：你可以通过设置 `SpatialAudioParams` 中的 `enableDoppler` 参数开启多普勒音效，在声源和接收方发生高速相对位移的情况下 (比如赛车游戏场景) ，接收方会体验到明显的音调变化。
+- 多普勒音效：你可以通过设置 `SpatialAudioParams` 中的 `enable_doppler` 参数开启多普勒音效，在声源和接收方发生高速相对位移的情况下 (比如赛车游戏场景) ，接收方会体验到明显的音调变化。
 - 耳机均衡器：你可以通过调用 `setHeadphoneEQPreset` 方法使用预设的耳机均衡效果，以改善耳机的听感。
 
 **7. 耳机均衡器**
 
 该版本新增 `setHeadphoneEQParameters` 方法，用于调节耳机均衡器的低频和高频参数，主要应用于空间音效场景。如果在调用 `setHeadphoneEQPreset` 后仍未达到预期的耳机均衡效果，你可以调用该方法进行调节。
```
Comment (Cilla-luodan) [inline markdown/Flutter NG/Misc/Electron 410 发版说明/视频.md:60]: 这里的耳机均衡器跟上面的合并到一起，因为 native 是分两个版发的，在框架是一次性合入

File: markdown/RTC 4.x/zh-CN/release_rn_ng.md [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+如需停止多路摄像头采集，需要先调用 `stopSecondaryCameraCapture` 停止第二个摄像头采集，然后调用 `enableMultiCamera` 并将 `enabled` 设置为 `false`。
+
+**8. 版权音乐**
+
+为解决直播、语聊房、在线 K 歌房等场景下歌曲的应用版权问题，该版本新增版权音乐相关 API。你可以通过调用 `MusicContentCenter` 类、`MusicPlayer` 类、`MusicContentCenterEventHandler` 类下的相关 API 实现在实时互动场景中播放版权音乐以及相关功能，例如检索音乐资源、获取音乐榜单及榜单详情、预加载及播放音乐资源、下载歌词及海报等功能。你还可以参考[在线 K 歌房](https://docs.agora.io/cn/online-ktv/ktv_overview)来体验搭配了演唱评分、美声音效等一系列功能的线上 K 歌场景化解决方案。
+
+**9. 耳机均衡器**
+
+该版本新增 `setHeadphoneEQParameters` 方法，用于调节耳机均衡器的低频和高频参数，主要应用于空间音效场景。如果在调用 `setHeadphoneEQPreset` 后仍未达到预期的耳机均衡效果，你可以调用该方法进行调节。
+
```
Comment (jinyuagora) [inline markdown/RTC 4.x/zh-CN/release_rn_ng.md:69]: 好像少一个 section - 已编码的视频观测器

File: markdown/RTC 4.x/zh-CN/release_rn_audio_ng.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+**1. 蓝牙权限 (Android)**
+
+为简化集成步骤，自该版本起，SDK 支持你在不添加 `BLUETOOTH_CONNECT` 权限的情况下，也能让 Android 用户正常使用蓝牙。
+
+**2. 跨频道媒体流转发**
+
+该版本对 `updateChannelMediaRelay` 方法做了如下优化：
+
+- 4.1.0 版本前：如果服务器内部原因导致目标频道更新失败，SDK 返回错误码 `relayEventPacketUpdateDestChannelRefused (8)`，你需要重新调用 `updateChannelMediaRelay` 方法。
```
Comment (jinyuagora) [inline markdown/RTC 4.x/zh-CN/release_rn_audio_ng.md:77]: relayEventPacketUpdateDestChannelRefused (8) -> RelayEventPacketUpdateDestChannelRefused (8)
大写 悲

File: markdown/RTC 4.x/zh-CN/release_rn_audio_ng.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+- `ChannelMediaOptions`：新增 `isAudioFilterable`
+- `SpatialAudioParams`：新增 `enable_doppler`
+- `leaveChannelEx`：新增 `options`
+- `LocalAccessPointConfiguration`：新增 `advancedConfig`
+- `onClientRoleChanged`：新增 `newRoleOptions`
+
+**废弃**
+
+- `onApiCallExecuted`
+- `ChannelMediaRelayEvent`：废弃 `relayEventPacketUpdateDestChannelRefused (8)`
```
Comment (jinyuagora) [inline markdown/RTC 4.x/zh-CN/release_rn_audio_ng.md:161]: 同上

File: markdown/RTC 4.x/zh-CN/release_rn_audio_ng.md [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+
+- 支持两人及两人以上合唱。
+- 每位歌手相互独立。一位歌手出现问题或退出合唱，其他歌手还可以继续合唱。
+- 极低延时体验。每位歌手可以实时听到彼此的歌声，观众也可以实时听到每位歌手。
+
+该版本新增 `AudioScenarioChorus` 枚举来设置极低延时。使用该枚举后，在网络条件良好的情况下，用户可以体验到极低延时的实时合唱。
+
+**7. 增强的频道管理**
+
+为满足各类业务场景对频道管理的需求，该版本在 `ChannelMediaOptions` 结构体中新增了如下功能：
```
Comment (jinyuagora) [inline markdown/RTC 4.x/zh-CN/release_rn_audio_ng.md:231]: 扫到一个上个版本的问题
应该只有 cpp 叫结构体？

File: markdown/RTC 4.x/release_rn_ng.md [hunk 6]
```diff
@@ -47,19 +47,15 @@ If you want to modify this default behavior, you can call `setDualStreamMode` an
...
+**7. Multiple cameras for video capture (iOS)**
 
 This release supports multi-camera video capture. You can call `enableMultiCamera` to enable multi-camera capture mode, call `startSecondaryCameraCapture` to start capturing video from the second camera, and then publish the captured video to the second channel.
 
 To stop using multi-camera capture, you need to call `stopSecondaryCameraCapture` to stop the second camera capture, then call `enableMultiCamera` and set `enabled` to `false`.
 
-**7. Headphone equalization effect**
-
-This release adds the `setHeadphoneEQParameters` method, which is used to adjust the low- and high-frequency parameters of the headphone EQ. This is mainly useful in spatial audio scenarios. If you cannot achieve the expected headphone EQ effect after calling `setHeadphoneEQPreset`, you can call `setHeadphoneEQParameters` to adjust the EQ.
-
```
Comment (jinyuagora) [inline markdown/RTC 4.x/release_rn_ng.md:62]: 同中文 是不是少一个 section
**8. Encoded video frame observer**
This release adds the `setRemoteVideoSubscriptionOptions` and `setRemoteVideoSubscriptionOptionsEx` methods. When you call the `registerVideoEncodedFrameObserver` method to register a video frame observer for the encoded video frames, the SDK subscribes to the encoded video frames by default. If you want to change the subscription options, you can call these new methods to set them.
For more information about registering video observers and subscription options, see the [API reference](./API%20Reference/rn_ng/API/toc_video_observer.html#api_imediaengine_registervideoencodedframeobserver).

**Pull request merged**